components/sync15/src/request.rs

```diff
@@ -1153,6 +1196,8 @@ mod test {
         );
     }
 
+    // Tests running multiple post queues on multiple batches,
+    // tests it specifically changing bytes
```

Comment:
Commenting these tests is great, but can you be more specific about what they do, either in the tests themselves, or in these comments. Typically the name of the test itself should be a good indication of what it is testing. Then, comments in the test should describe _how_ and _why_ the test is doing what it's doing.